appjs/templates/project.ejs

```diff
@@ -1,6 +1,14 @@
 <div class="row m-page-heading">
   <div class="col-xs-12">
-    <h3><% if ( model.isNew() ) { %>New Project<% } else { %><%=model.get( 'title' ) %><% } %></h3>
+    <h3>
+        <% if ( model.isNew() && (copyProject == false) ) {
+          %>New Project<%
+        } else if ( copyProject ) {
+          %>New <%= model.get( 'title' ) %><%
```

Comment:
It appends the title after changing the title to `Copy of <%= model.get( 'title' ) %>`, so the final output is `New Copy of <old title>`.